csm_web/scheduler/management/commands/patchattendances.py

```diff
@@ -18,14 +18,10 @@ def handle(self, *args, **options):
             week_start = today - timedelta(days=today.weekday())
```

Comment:
This logic is incorrect. `week_start` to `week_end` will definitely be longer than a week. Say today is Sunday (day 6 in Python) 9/22, then `week_start` is `today` minus 6 days, which is 9/16. This much is correct. However we then set `week_end = today + timedelta(days=7)`, which is 9/29. Clearly 9/16 - 9/29 is longer than one week.

Reply:
should've been week_start + timedelta 7, oops